test/kotlin/integration/StatFlushIntegrationTest.kt

```diff
@@ -0,0 +1,76 @@
+package test.kotlin.integration
+
+import io.envoyproxy.envoymobile.Element
+import io.envoyproxy.envoymobile.Engine
+import io.envoyproxy.envoymobile.EngineBuilder
+import io.envoyproxy.envoymobile.LogLevel
+import io.envoyproxy.envoymobile.engine.JniLibrary
+import java.util.concurrent.CountDownLatch
+import java.util.concurrent.TimeUnit
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.After
+import org.junit.Test
+
+class StatFlushIntegrationTest {
+  private var engine: Engine? = null
+
+  init {
+    JniLibrary.loadTestLibrary()
+  }
+
+  @After
+  fun teardown() {
+    engine?.terminate()
+    engine = null
+  }
+
+  @Test
+  fun `concurrent flushes with histograms`() {
```

Comment:
im assuming the goal for this test case is to verify it's safe to concurrently flushing, could you confirm?

Reply:
It's to verify that we can have multiple stats flushes "queued up" on the dispatcher at the same time, regression for the issue we ran into previously. The contains thing is just to ensure that Envoy has booted up before we start hitting it with flushes

Reply:
👍